plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/impl/editor/AbstractEditor.java

```diff
@@ -180,6 +183,44 @@ public ContentAssistant openContentAssistant() {
         return new ContentAssistant(caw.getContentAssistTable());
     }
 
+	public ContentAssistant openContentAssistant(ContentAssistantEnum assistantType) {
+		return openContentAssistant(assistantType.getLabel());
+	}
+
+    public ContentAssistant openContentAssistant(String assistantLabel) {
+    	activate();
+    	log.info("Open editor's " + assistantLabel + " content assistant");
+        AbstractWait.sleep(TimePeriod.SHORT);
+        Shell[] shells1 = ShellLookup.getInstance().getShells();
+        
+        new ShellMenu("Edit", "Content Assist", assistantLabel).select();
```

Comment:
is I pass non-existing assistantLabel, I should get some meaningful exception like "Content assistant "+assistantLabel+ " does not exist". 

or another case is if assistantLabel is found but is disabled - "Content assistant "+assistantLabel+ " is disalbed".